doc/connectivity/bluetooth/api/audio/bluetooth-le-audio-arch.rst

```diff
@@ -59,6 +59,334 @@ GAF has been implemented in Zephyr with the following structure.
+   | CAP Initiator                     | M            |               | M                    |                        | M                      |                          |
+   +-----------------------------------+--------------+---------------+----------------------+------------------------+------------------------+--------------------------+
+   | CAP Acceptor                      |              | M             |                      | M                      |                        | M                        |
+   +-----------------------------------+--------------+---------------+----------------------+------------------------+------------------------+--------------------------+
+   | CAP Commander                     | M            | O             | M                    | O                      | O                      | O                        |
+   +-----------------------------------+--------------+---------------+----------------------+------------------------+------------------------+--------------------------+
+   | BAP Broadcast Source              |              |               |                      |                        | M                      |                          |
+   +-----------------------------------+--------------+---------------+----------------------+------------------------+------------------------+--------------------------+
+   | BAP Broadcast Sink                |              |               |                      |                        |                        | M                        |
+   +-----------------------------------+--------------+---------------+----------------------+------------------------+------------------------+--------------------------+
+   | BAP Audio Source, Unicast Client  | M            |               | M                    |                        |                        |                          |
+   +-----------------------------------+--------------+---------------+----------------------+------------------------+------------------------+--------------------------+
+   | BAP Audio Sink, Unicast Client    | M            |               |                      |                        |                        |                          |
+   +-----------------------------------+--------------+---------------+----------------------+------------------------+------------------------+--------------------------+
+   | BAP Audio Source, Unicast Server  |              | C.1           |                      | M                      |                        |                          |
+   +-----------------------------------+--------------+---------------+----------------------+------------------------+------------------------+--------------------------+
+   | BAP Audio Sink, Unicast Server    |              | C.1           |                      |                        |                        |                          |
```

Comment:
Hmm, the Audio Sink and Audio Source are not really profile roles like the others. 

Suggest to remove the references to those type of roles and make these conditionals